### ISSUE TITLE
Bug combattimento Shulong

### ISSUE THREAD

**Issue by Absol99** [fuji97/pokemon-xenoverse-tracker#472]
Non mi compare lo scaleon di Oleandro e inoltre mentre combatto Shulong mi si è bloccato sulla schermata di selezione Pokèmon e non posso più vedere nulla e nemmeno cambiare Pokèmon

**Comment by SkyAxel** [comment]
E' successo anche a me, ho risolto componendo una squadra di sei membri, purtroppo si è presentato un altro bug: dopo averlo sconfitto torna primo in squadra ma cancella un pokemon che era presente in essa, andando a controllare il box non era presente (quindi è stato cncellato) e dopo qualche secondo nella schermata dei box il gioco è crashato.

**Comment by Absol99** [comment]
Quindi in questo caso compare dopo che tutti gli altri sono stati socnfitti?

**Comment by SkyAxel** [comment]
No, non compariva comunque. Ma ho scoperto il metodo per risolvere nella discussione dello stesso problema ma riguardante Trishout: in pratica basta disinstallare il gioco (ricorda di mettere i dati di gioco in un altra cartella) e installarlo nuovamente fino alla 1.0.1 e di installare la 1.0.2 solo dopo averlo sconfitto. Nella 1.0.1 infatti il bug non si presenta.
